src/prism.c

```diff
@@ -5821,6 +5824,18 @@ pm_parser_local_add(pm_parser_t *parser, pm_constant_id_t constant_id) {
     }
 }
 
+/**
+ * Add a constant id to the local table of the current scope.
```

Comment:
This comment doesn't make a lot of sense in this context.

Reply:
Good catch! https://github.com/ruby/prism/pull/1945